{"content": {"suites": {"0": {"status": {"total_pass": 4, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_database_connection", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_data_insertion", "rerun": "0"}, "2": {"status": "FAIL", "message": "   NameError: name 'get_db_connection' is not defined\n", "test_name": "test_fetch_users", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_update_data", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_delete_data", "rerun": "0"}}, "suite_name": "tests/test_hw_29_postgres.py"}}}, "date": "December 04, 2024", "start_time": 1733340227.7905338, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "4", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "5"}